.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+# This configuration file tells Dependabot which
+# package ecosystems to update and where the package manifests are located.
+# https://docs.github.com/en/enterprise-cloud@latest/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates
+
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  # https://github.com/dependabot/dependabot-core/pull/6189
+  - package-ecosystem: "github-actions"
+    directory: "/.github/workflows"
```

Comment:
is this path correct? I noticed it is different than s2n-quic: https://github.com/aws/s2n-quic/blob/b862ad982859f03d7e45f5d6291f749697a04a0f/.github/dependabot.yml#L10

Reply:
Picked this based on comments in the linked [doc](https://github.com/dependabot/dependabot-core/pull/6189) and the [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directory), since we don't have any action.yml files.